style.scss

```diff
@@ -0,0 +1,161 @@
+@import "./variables";
+@import "./reset";
+
+%price-common {
+  font-family: "Source Sans Pro";
+  font-style: normal;
+  font-weight: 400;
+}
+%plan-header-common {
+  font-family: "Montserrat";
+  font-style: normal;
+  line-height: 54px;
+}
+
+.plan-row {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 39.1rem);
+  justify-content: center;
+  padding: 75px;
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-gap: 55px;
+  }
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(auto-fit, 340px);
+    grid-gap: 55px;
+    display: grid;
+  }
+  .plan-col {
+    position: relative;
+    .price {
+      background-color: $bgPrice;
+      position: absolute;
+      text-align: center;
+      padding-top: 35px;
+      border-radius: 50%;
+      height: 120px;
```

Comment:
Trường hợp fix cứng như này. nếu price có giá trị lớn, thì UI có bị bug hay lỗi gì không

Reply:
nếu mà mình chỉnh to quá thì nó bị to hơn, em sửa thành rem rồi ạ